ui/src/kapacitor/components/RuleHandlers.js

```diff
@@ -133,9 +133,14 @@ class RuleHandlers extends Component {
     } = this.props
     const {handlersOnThisAlert, selectedHandler} = this.state
 
-    const handlers = _.map([...DEFAULT_HANDLERS, ...handlersFromConfig], h => {
-      return {...h, text: h.type}
-    })
+    const handlers = _.reverse(
+      _.sortBy(
+        _.map([...DEFAULT_HANDLERS, ...handlersFromConfig], h => {
+          return {...h, text: h.type}
+        }),
+        ['enabled']
+      )
+    )
```

Comment:
pls investigate teh lodash chaining api
don't forget to call value as the last method
👍 
`_.map([]).sortBy(['enabled']).reverse().value()`

Reply:
decided to insert a separator between enabled and not enabled handlers, so changed the logic here. Take a look ! :)